### ISSUE TITLE
Problemas con el phpmyadmin

### ISSUE THREAD

**Issue by ENZOPERU** [phpmyadmin/phpmyadmin#17841]
Buenas noches, resulta que existe un error en el phpMyadmin, al momento de generar una consulta, con los criterios de: ASCENDENTE y  DESCENDENTE, tal como aparece en la pantalla: 
![Captura](https://user-images.githubusercontent.com/116937927/198816748-3a298024-9b1d-418f-af28-73f69ca327c3.PNG)

**Comment by kamil-tekiela** [comment]
Por favor, traduzca al inglés si es posible.

**Comment by liviuconcioiu** [comment]
Indeed, the query doesn't including the sorting ASC or DESC. Both QA_5_2 and master are affected.

https://user-images.githubusercontent.com/25424343/198828797-1812f853-6d74-466d-9605-7765ff4f76e8.mp4

**Comment by williamdes** [comment]
I am looking into it. I have a commit to fix a painful behavior:
- click on "criteria"
- tick the box
- it closes the aera 🤯

After 66a7aa4b232838481ef85d9ac385eccdfd1bf8fa this will not be anymore the case

**Comment by williamdes** [comment]
I am trying to reproduce this issue on QA_4_9 and it seems like it did not work

**Comment by williamdes** [comment]
Seems have been added into 4.8 but I can not get it working.
![image](https://user-images.githubusercontent.com/7784660/198833338-9cbf7cc9-7ad1-443f-a1f6-99f33ca60502.png)

**Comment by Arpit-Gulgulia** [comment]
Is this issue still open to working on?

**Comment by williamdes** [comment]
> Is this issue still open to working on?

Yes, please go ahead. I think there is some digging to do because I did not find code related to making it work nor did I did find how it was supposed to work on the first version it was released on

**Comment by Arpit-Gulgulia** [comment]
@williamdes, thanks for the heads up. Unfortunately, I am not getting much time after my office hours. Will work on it when I get some time Thanks.

**Comment by danielTiringer** [comment]
@williamdes I came across this issue while using myadmin, and want to fix it.
Personally, I would separate the "sort" and "criteria", as they aren't related and should be usable separately.
I would move "sort" under another checkbox, and apply the appropriate sort radio button's value only if the box is checked.
Also, is it ok if I add a `watch: webpack --watch` script to the package.json for js/ts rebuild on changes, or that shouldn't be mixed in here?

**Comment by DPinheiro24** [comment]
Hello everyone, saw this issue marked as "good first issue". I have cloned the most recent version of the repo and can confirm this bug is still present. I believe I might be able to apply a fix. Is it alright if I work on this issue?

**Comment by liviuconcioiu** [comment]
Feel free to open a PR. You should use `QA_5_2` branch.
